utilities/test_suite/HOST/runAudioTests.py

```diff
@@ -0,0 +1,294 @@
+    parser.add_argument('--preserve_output', type = int, default = 1, help = "preserves the output of the program - (0 = override output / 1 = preserve output )" )
+    parser.add_argument('--batch_size', type = int, default = 1, help = "Specifies the batch size to use for running tests. Default is 1.")
+    parser.add_argument('--qa_mode', type = int, default = 0, help = "Run with qa_mode? Outputs from tests will be compared with golden outputs - (0 / 1)", required = False)
+
+    args = parser.parse_args()
+
+    # check if the folder exists
+    validate_path(args.input_path)
+
+    # validate the parameters passed by user
+    if ((args.case_start < 0 or args.case_start > 0) or (args.case_end < 0 or args.case_end > 0)):
+        print("Starting case# and Ending case# must be in the 0:0 range. Aborting!")
+        exit(0)
+    elif args.case_end < args.case_start:
+        print("Ending case# must be greater than starting case#. Aborting!")
+        exit(0)
+    elif args.test_type < 0 or args.test_type > 1:
+        print("Test Type# must be in the 0 / 1. Aborting!")
+        exit(0)
+    elif args.case_list is not None and args.case_start > 0 and args.case_end < 0:
+        print("Invalid input! Please provide only 1 option between case_list, case_start and case_end")
+        exit(0)
+    elif args.num_runs <= 0:
+        print("Number of Runs must be greater than 0. Aborting!")
+        exit(0)
+    elif args.batch_size <= 0:
+        print("Batch size must be greater than 0. Aborting!")
+        exit(0)
+    elif args.qa_mode < 0 or args.qa_mode > 1:
+        print("QA mode must be in the 0 / 1. Aborting!")
+        exit(0)
+    elif args.preserve_output < 0 or args.preserve_output > 1:
+        print("Preserve Output must be in the 0/1 (0 = override / 1 = preserve). Aborting")
+        exit(0)
+
+
+    if args.case_list is None:
+        args.case_list = range(args.case_start, args.case_end + 1)
+        args.case_list = [str(x) for x in args.case_list]
+    else:
+        for case in args.case_list:
+            if int(case) != 0:
+                 print("The case# must be 0!")
+                 exit(0)
+
+    return args
+
+args = rpp_test_suite_parser_and_validator()
+srcPath = args.input_path
+caseStart = args.case_start
+caseEnd = args.case_end
+testType = args.test_type
+caseList = args.case_list
+numRuns = args.num_runs
+qaMode = args.qa_mode
+preserveOutput = args.preserve_output
+batchSize = args.batch_size
+bitDepth = 2
+outFilePath = " "
+
+if preserveOutput == 0:
+    validate_and_remove_folders(cwd, "OUTPUT_AUDIO_HOST")
+    validate_and_remove_folders(cwd, "QA_AUDIO_RESULTS_HOST")
+    validate_and_remove_folders(cwd, "OUTPUT_AUDIO_PERFORMANCE_LOGS_HOST")
+
+if(testType == 0):
+    if qaMode:
+        outFilePath = os.path.join(os.path.dirname(cwd), 'QA_AUDIO_RESULTS_HOST_' + timestamp)
+    else:
+        outFilePath = os.path.join(os.path.dirname(cwd), 'OUTPUT_AUDIO_HOST_' + timestamp)
```

Comment:
Please remove the usage of this OUTPUT_AUDIO_HOST

Reply:
discussed about this change and decided to have the folder name as OUTPUT_AUDIO_HOST for now